db/migrate/20231004120309_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
```

Comment:
- [ ] [optional] @SanaTameem I would recommend changing the datatype from string to text, this is because text allows for a large input compared to string and would be suitable in this instance.